libs/wire-api/src/Wire/API/Event/UserGroup.hs

```diff
@@ -0,0 +1,42 @@
+{-# LANGUAGE StrictData #-}
+
+-- This file is part of the Wire Server implementation.
+--
+-- Copyright (C) 2025 Wire Swiss GmbH <opensource@wire.com>
+--
+-- This program is free software: you can redistribute it and/or modify it under
+-- the terms of the GNU Affero General Public License as published by the Free
+-- Software Foundation, either version 3 of the License, or (at your option) any
+-- later version.
+--
+-- This program is distributed in the hope that it will be useful, but WITHOUT
+-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+-- details.
+--
+-- You should have received a copy of the GNU Affero General Public License along
+-- with this program. If not, see <https://www.gnu.org/licenses/>.
+
+module Wire.API.Event.UserGroup where
```

Comment:
take this out as we don't use it?  no strong opinion.

Reply:
> take this out as we don't use it? no strong opinion.
The copyright header should be in.

Reply:
i didn't mean the (C) stuff, but the entire module.  i don't think it's used anywhere (yet).